bindings/rust/aws-lc-sys-template/src/lib.rs

```diff
@@ -1,22 +1,24 @@
-#![allow(non_upper_case_globals)]
-#![allow(non_camel_case_types)]
-#![allow(non_snake_case)]
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
 
-// populated by cmake
-${INCLUDES}
+mod bindings;
+pub use bindings::*;
 
+#[allow(non_snake_case)]
 pub fn ERR_GET_LIB(packed_error: u32) -> i32 {
     unsafe { ERR_GET_LIB_RUST(packed_error) }
 }
 
+#[allow(non_snake_case)]
 pub fn ERR_GET_REASON(packed_error: u32) -> i32 {
     unsafe { ERR_GET_REASON_RUST(packed_error) }
 }
 
+#[allow(non_snake_case)]
 pub fn ERR_GET_FUNC(packed_error: u32) -> i32 {
     unsafe { ERR_GET_FUNC_RUST(packed_error) }
 }
 
 pub fn init() {
-    unsafe { CRYPTO_library_init(); }
+    unsafe { CRYPTO_library_init() }
 }
```

Comment:
Is this just an alias? Can this function be called multiple times? On multiple concurrent threads? It's probably best to keep it unsafe if it's not truly idempotent and synchronized.

Reply:
According to [its documentation](https://github.com/awslabs/aws-lc/blob/610e2f58a8596a84be3207d433a2ca3b0c2cb63b/include/openssl/crypto.h#L41):
> // CRYPTO_library_init initializes the crypto library. It must be called if the
> // library is built with BORINGSSL_NO_STATIC_INITIALIZER. Otherwise, it does
> // nothing and a static initializer is used instead. It is safe to call this
> // function multiple times and concurrently from multiple threads.

Reply:
We don't build AWS-LC with that flag, so this function could probably be removed altogether.